mmv1/templates/terraform/pre_update/alloydb_instance.go.erb

```diff
@@ -0,0 +1,25 @@
+// If the SSL mode isn't specified in the update, retain the SSL mode that's already present
```

Comment:
What is the reason to retain the old SLL mode? I think it is better to make SSL mode default to `"ENCRYPTED_ONLY"` to match the create method.

Reply:
This is to maintain backwards compatibility as there are resources that don't have the `clientConnectionField` even (due to being created before feature was launched), so it is set to NULL. Plus, in the AlloyDB API we do not update fields unless the customer explicitly asks to update it themselves. We don't want to break these customer's existing configuration, and cause confusion for them due to this unexpected behavior.